x-pack/plugins/observability_solution/apm/public/hooks/use_eco_tour.tsx

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { useLocalStorage } from './use_local_storage';
+
+type TourState = 'shownBefore' | 'isActive';
+
+const INITIAL_STATE: Record<TourState, boolean> = {
+  shownBefore: false,
```

Comment:
What does `showBefore` mean? Usually on a tour state we have the current step.

Reply:
We don't want to show the modal and the tour every time the user switches to the new experience, only the first time but the naming is bad, what about `isEcoTourSeen` ?